rust/windows-client/src-tauri/build.rs

```diff
@@ -1,3 +1,18 @@
 fn main() {
-    tauri_build::build()
+    let win = tauri_build::WindowsAttributes::new().app_manifest(WINDOWS_MANIFEST);
+    let attr = tauri_build::Attributes::new().windows_attributes(win);
+    tauri_build::try_build(attr).expect("build.rs");
```

Comment:
the `expect` message is not very clear

Reply:
Removed in 839122a2, I wasn't sure if `anyhow` could be used in build.rs, but it can be.
I figured if this error happens at build time it'll show the line number, and there's nothing I can add to that. But letting anyhow bubble tauri's error should provide more info.